src/main/scala/xiangshan/frontend/ITTAGE.scala

```diff
@@ -280,11 +280,14 @@ class ITTageTable
   // only when ctr is null
   update_wdata.target := Mux(io.update.alloc || ctr_null(old_ctr), update_target, io.update.old_target)
 
-  val newValidArray = VecInit(validArray.asBools)
-  when (io.update.valid) {
-    newValidArray(update_idx) := true.B
-    validArray := newValidArray.asUInt
-  }
+  // val newValidArray = VecInit(validArray.asBools)
+  // when (io.update.valid) {
+  //   newValidArray(update_idx) := true.B
+  //   validArray := newValidArray.asUInt
+  // }
+  // val validArrayTest =  RegInit(0.U(nRows.W));
+  when (io.update.valid) { validArray := UIntToOH(update_idx) | validArray }
```

Comment:
follow the code style as other lines in this file please

Reply:
ok